subprojects/gradle/cthulhu/src/main/kotlin/com/avito/plugin/CthulhuPluginApi.kt

```diff
@@ -0,0 +1,12 @@
+package com.avito.plugin
+
+import com.avito.kotlin.dsl.typedNamed
+import org.gradle.api.tasks.TaskContainer
+import org.gradle.api.tasks.TaskProvider
+
+internal const val markReportAsSourceTaskName = "markReportAsSourceForTMSTask"
```

Comment:
Naming question: i use both TMS and Cthulhu; should it be the one?
Integration is Cthulhu specific i suppose

Reply:
I vote for one name

Reply:
ok, i'll rename all to TMS